detekt-rules-documentation/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/documentation/AbsentOrWrongFileLicense.kt

```diff
@@ -10,7 +10,9 @@ import io.gitlab.arturbosch.detekt.api.Severity
 import org.jetbrains.kotlin.psi.KtFile
 
 /**
- * This rule will report every Kotlin source file which doesn't have required license header.
+ * This rule will report every Kotlin source file which doesn't have the required license header.
+ * Each Kotlin source file is checked, whether the header starts with the read text from the passed file in the
```

Comment:
```suggestion
 * Each Kotlin source file is checked whether the header starts with the read text from the passed file in the
```

Not sure if the comma is needs to be removed here.